test/libraries/MultiSend.spec.ts

```diff
@@ -174,5 +182,62 @@ describe("MultiSend", async () => {
                 ),
             ).to.be.eq("0x" + "baddad".padEnd(64, "0"));
         });
+
+        it("can bubble up revert message on call", async () => {
+            const { delegateCaller, multiSend, mock } = await setupTests();
+
+            const trigguerCalldata = "0xbaddad";
+            const errorMessage = "Some random message";
+
+            await mock.givenCalldataRevertWithMessage(trigguerCalldata, errorMessage);
+            const rawError = buildRawError(errorMessage);
+
+            const txs: MetaTransaction[] = [
+                {
+                    to: mock.address,
+                    value: 0,
+                    data: trigguerCalldata,
+                    operation: 0,
+                },
+            ];
+            const { data } = buildMultiSendSafeTx(multiSend, txs, 0);
+
+            const { success, returnData } = await delegateCaller.callStatic.makeDelegatecal(multiSend.address, data);
```

Comment:
Is there a reason for using callStatic instead of executing a transaction calling the contract?

Reply:
> Is there a reason for using callStatic instead of executing a transaction calling the contract?

The callstatic is to get the returndata that is not given in a transaction, I thought in bubling up the error message in this mock contract but I opted this code for the sake of simplicity.

Let me know how you like best